### ISSUE TITLE
[feature request] Make it easier to get to conda page from a tool's requirements

### ISSUE THREAD

**Issue by hexylena** [galaxyproject/galaxy#17432]
These are all conda dependencies. If the server is configured to resolve a dependency via conda, I'd be really neat if these link out to e.g. https://anaconda.org/bioconda/bioconductor-goseq or maybe https://anaconda.org/bioconda/bioconductor-goseq/files?version=1.50.0 since there's no version-specific index page. i recognise that the exact version run is potentially unknown until the job is run, but it still could be nice, especially since the "See Details" button goes to https://galaxyproject.org/tools/requirements/ , which is *technically* details, but it isn't the details I was expecting/wanting. For now I can copy paste into the last component of `https://anaconda.org/bioconda/$package` and that's ok.

![image](https://github.com/galaxyproject/galaxy/assets/458683/d02bb6c7-ee40-4baf-adc9-17240a2ddad1)

**Comment by nsoranzo** [comment]
That seems something that should be done in the "Job Dependencies" section of the job details instead, since, as you mention, how this requirements are resolved is not determined until after the job has run.

**Comment by bgruening** [comment]
@hexylena couldn't we use the xref annotations for this: https://github.com/galaxyproject/tools-iuc/blob/dfe519563a762fc72953e4754f76effe5e16cfec/tools/heinz/bum.xml#L5

The requirements are not strictly linked to conda and it is probably hard to distinguish between primary and secondary "dependencies". With the xref, we can get those main citing elements.

**Comment by hexylena** [comment]
> That seems something that should be done in the "Job Dependencies" section of the job details instead, since, as you mention, how this requirements are resolved is not determined until after the job has run.

yes but that's more clicks to get to, and what if I want to link that information without finding datasets compatible for running that tool? 

> The requirements are not strictly linked to conda

*de jure*, no, but, *de facto* they are, for the vast majority of tools folks care about though, right? I don't remember the last time I used a non-conda dependency, and even for containers they're biocontainers based on conda. So, if it's using a container, linking to the anaconda page is still *useful*.

> hard to distinguish between primary and secondary

yes, of course. in the case where you need the primary dependency of course xref would be better. Here I was think we could add something automatic, based on dependencies, rather than manual annotation (given how long it will take us to get good EDAM terms coverage).

But xrefs also aren't often used to link to dependencies, rather to point to bio.tools (very hit and miss whether it is helpful or not). 

I didn't know xref supports bioconductor, for that sub-case, it's even better, and saves even more clicks. We should be sure all bioconductor tools are properly annotated, that will improve that sub-case going forward!

**Comment by bgruening** [comment]
> I didn't know xref supports bioconductor, for that sub-case, it's even better, and saves even more clicks. We should be sure all bioconductor tools are properly annotated, that will improve that sub-case going forward!

Yeah, we should. As usual, we have a lot of infrastructure pieces we just need to use them and update our fleet of tools to do so. I will include that in my review from now on.

**Comment by bgruening** [comment]
> I didn't know xref supports bioconductor, for that sub-case, it's even better, and saves even more clicks. We should be sure all bioconductor tools are properly annotated, that will improve that sub-case going forward!

Yeah, we should. As usual, we have a lot of infrastructure pieces we just need to use them and update our fleet of tools to do so. I will consider this in my reviews from now on.
